llama_index/llms/localai.py

```diff
@@ -37,11 +50,20 @@ class LocalAI(OpenAI):
 
     def __init__(
         self,
-        api_key: Optional[str] = DEFAULT_API_KEY,
-        api_base: Optional[str] = DEFAULT_API_BASE,
+        api_key: Optional[str] = LOCALAI_DEFAULTS["api_key"],
+        api_base: Optional[str] = LOCALAI_DEFAULTS["api_base"],
         **kwargs: Any,
     ) -> None:
         super().__init__(api_key=api_key, api_base=api_base, **kwargs)
+        warnings.warn(
```

Comment:
@Disiok this time I took your advice, and am doing a deprecation cycle ✅ 

We can keep the `localai` module around, and just have the `make_localai` function inside